org.eclipse.transformer/src/main/java/org/eclipse/transformer/action/impl/ClassActionImpl.java

```diff
@@ -1278,6 +1293,9 @@ private int transform(MutableConstantPool constants) throws TransformException {
 						if (outputString == null) {
 							transformCase = "Direct";
 							outputString = transformDirectString(inputString);
+                                                        if (outputString == null) {
+                                                            outputString = transformConstantString(inputString, inputName);
```

Comment:
Add a transformCase assignment, for example, "Direct per class".

Reply:
Thanks, done.